hacks/cp_enums.py

```diff
@@ -0,0 +1,31 @@
+# TEMPORARY HACK
```

Comment:
[JUST SAYING.. ] haha niceee! This is very handy. Since backend funcs and front-end web will be deployed separately, we can still regard them as independent components, like most of the web apps. Currently we can assume that the Enums.js for the front-end is the one to refer to since we are developing front-end first. There might be a concern in case that the front-end and back-end are under development in a different order by different developers. But it's perfectly fine if that would be the practice for this proj for now.

Reply:
😂Yeah exactly, when functions are deployed to the cloud, it only has access to sources inside the `functions` folder (only files in this folder are deployed), and frontend react can only import things from `src/`...

This is definitely not ideal at all, but I couldn't find or think of a proper way to make it happen. And I definitely do not want them to be different. So.. if you ever find/know a way, please do tell!